Extensions/TweenBehavior/tweenruntimebehavior.ts

```diff
@@ -424,10 +426,26 @@ namespace gdjs {
         duration: durationValue,
         easing: easingValue,
         step: function step(state) {
-          // @ts-ignore - objects are duck typed
-          that.owner.setScaleX(state.scaleX);
-          // @ts-ignore - objects are duck typed
-          that.owner.setScaleY(state.scaleY);
+          if(scaleFromCenterOfObject) {
+            const oldX = that.owner.getX();
+            const oldW = that.owner.getWidth();
+            // @ts-ignore - objects are duck typed
+            that.owner.setScaleX(state.scaleX);
+            const newX = oldX - (that.owner.getWidth() - oldW) / 2;
+            that.owner.setX(newX);
+
+            const oldY = that.owner.getY();
+            const oldH = that.owner.getHeight();
+            // @ts-ignore - objects are duck typed
+            that.owner.setScaleY(state.scaleY);
+            const newY = oldY - (that.owner.getHeight() - oldH) / 2;
+            that.owner.setY(newY);
```

Comment:
Sadly the computation for the center of an object are a bit more complex as some objects (like sprite) can have their center point moved.

I've just pushed on the `master` branch new methods for runtime object:
getCenterXInScene, getCenterYInScene, setCenterXInScene, setCenterYInScene (and setCenterPositionInScene).

This should enable this more concise and always valid code:
```suggestion
            const oldX = that.owner.getCenterXInScene();
            const oldY = that.owner.getCenterYInScene();
            // @ts-ignore - objects are duck typed
            that.owner.setScaleX(state.scaleX);
            // @ts-ignore - objects are duck typed
            that.owner.setScaleY(state.scaleY);
            that.owner.setCenterPositionInScene(oldX, oldY);
```

Let me know if it works :)

Reply:
Implemented, but I'm not sure if it will work as I don't know how to update the code that has the new functions without losing these ones.